ext/php7/coms.c

```diff
@@ -869,6 +903,8 @@ static void _dd_curl_send_stack(struct _writer_loop_data_t *writer, ddtrace_coms
         ddtrace_curl_set_connect_timeout(writer->curl);
 
         curl_easy_setopt(writer->curl, CURLOPT_UPLOAD, 1);
+        // SIGNALFX: use POST instead of PUT for the upload
+        curl_easy_setopt(writer->curl, CURLOPT_PUT, 1);
```

Comment:
Should this be behind `get_global_SIGNALFX_MODE`?

Reply:
Done.